bin/misc-functions.sh

```diff
@@ -478,8 +478,11 @@ __dyn_package() {
 	[ -z "${PORTAGE_BINPKG_TMPFILE}" ] && \
 		die "PORTAGE_BINPKG_TMPFILE is unset"
 	mkdir -p "${PORTAGE_BINPKG_TMPFILE%/*}" || die "mkdir failed"
+	COMPRESSION_COMMAND=$(PYTHONPATH=${PORTAGE_PYTHONPATH:-${PORTAGE_PYM_PATH}} \
```

Comment:
If all that we really need is COMPRESSION_COMMAND, then the helper script is overkill. We should just pass a variable from doebuild_environment function. The variable name must be prefixed with `PORTAGE_`.